cmd/teleirc.go

```diff
@@ -34,8 +37,8 @@ func main() {
 
 	settings, err := internal.LoadConfig(*flagPath)
 	if err != nil {
-		fmt.Println(err)
-		return
+		fmt.Fprintln(os.Stderr, err)
+		os.Exit(1)
```

Comment:
Aha – is this the exit code we talked about, in the context of systemd service files?

Reply:
Yep! Eventually we'll change it to use a proper logger, but this is better than nothing for the moment.